src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/ModifyQueueCapacityAction.java

```diff
@@ -30,7 +30,7 @@
 public class ModifyQueueCapacityAction implements Action {
 
   public static final String NAME = "modify_queue_capacity";
-  public static final int COOL_OFF_PERIOD_IN_SECONDS = 300;
+  public static final long COOL_OFF_PERIOD_IN_MILLIS = 300 * 1_000;
```

Comment:
can we set this in rca.conf and make this configurable

Reply:
Good idea, but out of scope for this commit. The only reason I changed this line is that we're going from s units to ms